README.md

```diff
@@ -119,7 +118,8 @@ Now it's just a matter of linking your `build.zig` target(s) to this library ins
 ```zig
 exe.linkLibC();
 exe.linkLibrary(sqlite);
-exe.addPackage(.{ .name = "sqlite", .path = "third_party/zig-sqlite/sqlite.zig" });
+exe.addPackage(.{ .name = "sqlite", .path = .{ .path = "third_party/zig-sqlite/sqlite.zig" } });
```

Comment:
this can actually be simplified to this:
```
exe.addPackagePath("sqlite", "third_party/zig-sqlite/sqlite.zig");
```

Reply:
@vrischmann 
Thank you for review! I didn't know this api.
Fix to use `addPackagePath` instead of `addPackage` in [be455e3](https://github.com/vrischmann/zig-sqlite/pull/101/commits/be455e3d97a308b37e316ed682862fc719139fa5).